samples/certs/Makefile

```diff
@@ -72,9 +78,15 @@ root-key.pem:
 ##<name>-certs:	generate Citadel certificates for <name>. Includes all PEM files needed.
 .PHONY: %-certs
 
+ifeq ($(WORKLOAD_CERTS),0)
```

Comment:
instead of a conditional, why not make this %-wl-certs? This simplifies the makefile by  extracting the changes for the virtual machine needed from the makefile into a separate call path.

Doing so would remove `WORKLOAD_CERTS` which is an unnecessary variable.